src/main/java/com/tokyoprogrammer/aqitracer/controller/AqitracerController.java

```diff
@@ -1,13 +1,47 @@
 package com.tokyoprogrammer.aqitracer.controller;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.stereotype.Component;
 
-@RestController
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.internal.shaded.guava.collect.ImmutableMap;
+import com.linecorp.armeria.server.annotation.Get;
+import com.linecorp.armeria.server.annotation.Param;
+import com.linecorp.armeria.server.annotation.Path;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Component
+@Slf4j
 public class AqitracerController {
 
-    @GetMapping("/healthCheck")
-    public String healthCheck() {
-        return "OK";
+    private enum AQI_PROVIDER {
+        WAQI,
+        KECO,
+        EUROPE,
+    };
+
+    private enum ETC_TYPE {
+        WALK,
+        CULTURE
+    };
+
+    @Get
+    @Path("/healthCheck")
+    public HttpResponse healthCheck() {
+        return HttpResponse.of(HttpStatus.OK, MediaType.JSON_UTF_8, "OK");
+    }
+
+    @Get
+    @Path("/api/v1/aqi")
+    public String getAqi(@Param("latlng") String latlng, @Param("provider") AQI_PROVIDER provider) {
+        return ImmutableMap.of("coordinate", latlng, "provider", provider).toString();
```

Comment:
We'd better have more discussion about parameters for this API.

Reply:
Yes. how about handling about API spec. for next issue?